CHANGELOG.md

```diff
@@ -1,6 +1,34 @@
 # Changelog
 
-## v1.18.0 - TBD
+## v1.19.0 - TBD
+
+## v1.18.0 - 2020-09-15
```

Comment:
We've never added a day number before, even when having multiple releases in the same month.